Cargo.toml

```diff
@@ -114,8 +114,7 @@ strum = { version = "0.25.0", default-features = false }
 getset = "0.1.2"
 hex-literal = "0.4.1"
 
-ursa = "0.3.7"
-aead = "0.3.2"
+aead = "0.5.2"
```

Comment:
move it into `crypto` crate

Reply:
Done. This also prompted me to refactor the error handling code in the symmetric encryption and iroha_p2p a bit.

Reply:
you didn't move it to iroha_crypto?

Reply:
Uhh, I think I misunderstood you before. `iroha_p2p` previously depended on `aead`, which is what I thought this was all about.

Now only `iroha_crypto` uses it and I removed it from the workspace `Cargo.toml` :+1: